src/redux/Reducer/routes.js

```diff
@@ -0,0 +1,74 @@
+import * as actions from "../Action/actiontypes.js";
+
+export function routesListReducer(state = { routes: [] }, action) {
+  switch (action.type) {
+    case actions.ROUTE_LIST_REQUEST:
+      return { loading: true, message: action.payload };
+    case actions.ROUTE_LIST_SUCCESS:
+      return { loading: false, routes: action.payload };
+    case actions.ROUTE_LIST_FAIL:
+      return { loading: false, error: action.payload };
+    default:
+      return state;
+  }
+}
+
+export function routestDetailsReducer(state = { route: {} }, action) {
+  switch (action.type) {
+    case actions.ROUTE_DETAILS_REQUEST:
+<<<<<<< HEAD
```

Comment:
Resolve these conflicts.

Reply:
Look like I missed this ones, I have resolved them